src/core/timing.ts

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
I have some parallel other functions for using `performance` here: https://github.com/quarto-dev/quarto-cli/blob/main/src/core/performance.ts (perhaps merge this into a single file)

Reply:
I completely missed that. Your functions are much nicer than mine, I'll just start building on them instead.